src/styles/main.scss

```diff
@@ -1,3 +1,77 @@
 body {
   margin: 0;
+  display: flex;
+  justify-content: center;
+}
+
+html {
+  font-family: Arial, sans-serif;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+$width-day: 100px;
+$gap: 1px;
+
+@mixin square ($width-day) {
+  width: $width-day;
+  height: $width-day;
+}
+
+@for $index from 1 through 31 {
+  .calendar__day:nth-child(#{$index})::before {
+    content: "#{$index}";
+    font-size: 30px;
+    line-height: 100px;
+  }
+}
+
+$first-day: 
+  "mon" 0,
+  "tue" 1,
+  "wed" 2,
+  "thu" 3,
+  "fri" 4,
+  "sat" 5,
+  "sun" 6;
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  max-width: 726px;
+  padding: 10px;
+  gap: $gap;
+
```

Comment:
.calendar is a block. So we can set internal and external propeties for the same class

Reply:
Sorry, I was wrong. Everything is Ok